### PR TITLE
VR > Tooltip > Add date and title.

### PR DESCRIPTION
**Pull request by juanfra**
🎫 https://central.tri.be/issues/137113
🎥 https://www.loom.com/share/9b347e730e614be085dadfcc78dedb1d

### CODE REVIEW

File: src/resources/postcss/views/full/month/_calendar-event.pcss [hunk 1]
```diff
@@ -89,4 +89,33 @@
...
+		display: inline-block;
+	}
+
+	.tribe-events-calendar-month__calendar-event-tooltip-datetime-featured {
+		margin-right: var(--spacer-0);
+	}
+
+	.tribe-events-calendar-month__calendar-event-tooltip-datetime-recurring-icon {
+		margin-left: var(--spacer-0);
+	}
```
Comment (paulmskim) [inline src/resources/postcss/views/full/month/_calendar-event.pcss]: these should be in skeleton

File: src/views/v2/month/calendar-body/day/calendar-events/calendar-event/tooltip/date.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+			aria-label="<?php esc_attr_e( 'Featured', 'the-events-calendar' ) ?>"
+			title="<?php esc_attr_e( 'Featured', 'the-events-calendar' ) ?>"
+		>
+		</em>
+	<?php endif; ?>
+	<time datetime="<?php echo esc_attr( $event->dates->start->format( 'H:i' ) ); ?>">
+		<?php echo esc_html( $event->dates->start->format( $time_format ) ); ?>
+	</time>
+	<?php if ( $display_end_date ) : ?>
+		<span class="tribe-events-calendar-month__calendar-event-tooltip-datetime-separator"> - </span>
```
Comment (paulmskim) [inline src/views/v2/month/calendar-body/day/calendar-events/calendar-event/tooltip/date.php]: should use the `dateTimeSeparator` option

File: tests/views_integration/Tribe/Events/Views/V2/Partials/Month_View/Calendar_Body/Day/Calendar_Events/Calendar_Event/Tooltip/DateTest.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+	protected $partial_path = 'month/calendar-body/day/calendar-events/calendar-event/tooltip/date';
+
+	/**
+	 * Test render with event
+	 */
+	public function test_render_with_event() {
+		$event = $this->get_mock_event( 'events/single/1.json' );
+		$this->assertMatchesSnapshot( $this->get_partial_html( [ 'event' => $event ] ) );
+	}
```
Comment (paulmskim) [inline tests/views_integration/Tribe/Events/Views/V2/Partials/Month_View/Calendar_Body/Day/Calendar_Events/Calendar_Event/Tooltip/DateTest.php]: can we also test a case where start time = end time?

File: src/views/v2/month/calendar-body/day/calendar-events/calendar-event/tooltip/date.php [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+ *
+ * @version TBD
+ *
+ * @var WP_Post $event The event post object with properties added by the `tribe_get_event` function.
+ *
+ * @see tribe_get_event() For the format of the event object.
+ */
+
+$time_format      = tribe_get_time_format();
+$display_end_date = $event->dates->start->format( 'H:i' ) !== $event->dates->end->format( 'H:i' );
```
Comment (paulmskim) [inline src/views/v2/month/calendar-body/day/calendar-events/calendar-event/tooltip/date.php]: same note as https://github.com/moderntribe/the-events-calendar/pull/2833#discussion_r345563585

File: src/views/v2/month/calendar-body/day/calendar-events/calendar-event/tooltip/date.php [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+		</em>
+	<?php endif; ?>
+	<time datetime="<?php echo esc_attr( $event->dates->start->format( 'H:i' ) ); ?>">
+		<?php echo esc_html( $event->dates->start->format( $time_format ) ); ?>
+	</time>
+	<?php if ( $display_end_date ) : ?>
+		<span class="tribe-events-calendar-month__calendar-event-tooltip-datetime-separator"> - </span>
+		<time datetime="<?php echo esc_attr($event->dates->end->format( 'H:i' ) ); ?>">
+			<?php echo esc_html( $event->dates->end->format( $time_format ) ); ?>
+		</time>
```
Comment (paulmskim) [inline src/views/v2/month/calendar-body/day/calendar-events/calendar-event/tooltip/date.php]: same note as https://github.com/moderntribe/the-events-calendar/pull/2833#discussion_r345563585

**Comment by juanfra** [comment]
@paulmskim this is ready for a new review. Thanks!

**Pull request merged**